shared/common-adapters/keyboard-avoiding-view.native.tsx

```diff
@@ -167,7 +172,9 @@ class KeyboardAvoidingView extends React.Component<Props, State> {
         },
       })
     }
-    this._setBottom(height)
+
+    // tablet modals are centered so we need to offset that, zero otherwise
+    this._setBottom(height - this._tabletLayoutHeight)
```

Comment:
on ipad the native modals are centered with some padding (different depending on orientation) vs on iphone the modal is at the bottom. We need to compensate for that else you get extra pushing up which is pretty bad on landscape ipad